force-app/main/default/classes/TestDataFactoryCountContactHandler.cls

```diff
@@ -0,0 +1,26 @@
+@isTest
+public class TestDataFactoryCountContactHandler {
+  
```

Comment:
Why this is separate class? can't we write this logic in the TestDataFactory class only?